rc/windowing/zellij.kak

```diff
@@ -0,0 +1,69 @@
+# https://zellij.dev/
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+provide-module zellij %{
+
+# ensure we're running under zellij
+evaluate-commands %sh{
+    [ -z "${client_opt_windowing_modules}" ] || [ -n "$ZELLIJ" ] || echo 'fail zellij not detected'
+}
+
+declare-option -hidden -docstring %{zellij run options (
+    -c, --close-on-exit            Close the pane immediately when its command exits
+        --cwd <CWD>                Change the working directory of the new pane
+    -d, --direction <DIRECTION>    Direction to open the new pane in
+    -f, --floating                 Open the new pane in floating mode
+    -h, --help                     Print help information
+    -n, --name <NAME>              Name of the new pane
+    -s, --start-suspended          Start the command suspended, only running after you first presses ENTER)}\
+    str zellij_run_options ""
+declare-option -hidden -docstring %{zellij run command} str zellij_run_cmd ""
+
+define-command -hidden -params 0 zellij-run %{
+    nop %sh{
+    zellij run ${kak_opt_zellij_run_options} -- $SHELL -i -c "${kak_opt_zellij_run_cmd}" < /dev/null > /dev/null 2>&1 &
```

Comment:
`connect.kak` spawns quite wild shell processes that will mess with the Zellij's pane name rendering. In #4850 I suggested setting it to nothing but that's probably negotiable.

Reply:
I prefer using the command which is the default so that we can know what that pane does especially when we have multiple jobs.

Reply:
let me if there is any real use which gives a trouble, thanks